service/controller/clusterapi/cluster_test.go

```diff
@@ -1,143 +0,0 @@
-package clusterapi
-
-import (
-	"net"
-	"testing"
-
-	versionedfake "github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
-	"github.com/giantswarm/aws-operator/service/network"
-	"github.com/giantswarm/micrologger/microloggertest"
-	apiextensionsclientfake "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
-	kubernetesfake "k8s.io/client-go/kubernetes/fake"
-	clientsetfake "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/fake"
-)
-
-func newTestClusterConfig() ClusterConfig {
```

Comment:
I dropped the tests here as I find them totally useless. Every time configuration changes the tests fail. Unit testing configuration is backwards and only increases development workload. Give me a shout if you have issues with this.